### PR TITLE
AdminBranding

### PR DESCRIPTION
**Pull request by agriffard**
Update OrchardCore.Admin docs.

Surround favicon by `zone "HeadMeta"` to put it in the head.

### CODE REVIEW

**Comment by agriffard** [comment]
As explained in https://docs.orchardcore.net/en/latest/docs/reference/modules/Admin/, you can change the AdminBranding by creating an Admin template:

image

**Review by MikeAlhayek — APPROVED** [review]

File: src/docs/reference/modules/Admin/README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ If you want to specify another prefix in the urls to access the admin section, y
 
 By default, OrchardCore logo and site name are displayed in the top navbar.
 
-You can change it by overriding 'AdminBranding' shape, either from a [custom admin theme](../../../guides/create-admin-theme/README.md) or using Admin Templates feature.
+You can change it by overriding 'AdminBranding' shape, either from a [custom admin theme](../../../guides/create-admin-theme/README.md) or using Admin Templates feature.
```
Comment (hishamco) [inline src/docs/reference/modules/Admin/README.md:20]: Remove extra whitespace at the end

**Review by hishamco — CHANGES_REQUESTED** [review]

File: src/docs/reference/modules/Admin/README.md (hunk 1 above)
Comment (agriffard) [reply src/docs/reference/modules/Admin/README.md:20]: That is on purpose: Double spaces force to the next sentence to be on a new line.

It currently lokks like this:
image

Comment (hishamco) [reply src/docs/reference/modules/Admin/README.md:20]: Strange!!

**Pull request merged**